styles/base.css

```diff
@@ -3,12 +3,12 @@
 }
 
 body {
-  background-color: #332f3c;
+  background-color: #1f1c26;
   display: grid;
   font-family: "Poppins", sans-serif;
   font-size: 0.75rem;
   grid-template-rows: 67px auto 724px;
```

Comment:
`724px` looks like it could break on some devices. Why not let it grow from it's content height?